packages/react-ui-core/src/Dropdown/Dropdown.js

```diff
@@ -70,6 +70,7 @@ export default class Dropdown extends Component {
   handleDocumentClick(event) {
     if (this.state.visible && !this.dropdown.contains(event.target)) {
       this.setState({ visible: false })
+      this.props.onVisibilityChange(false)
```

Comment:
I would update the test for `handleDocumentClick` too to check that the `onVisibilityChange` mocked func prop gets called too

Reply:
Added